src/containers/Podcast/PodcastListPage.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+// @ts-ignore
+import { OneColumn, Spinner } from '@ndla/ui';
+// @ts-ignore
+import Pager from '@ndla/pager';
+import { RouteComponentProps } from 'react-router';
+import { useLazyQuery } from '@apollo/client';
+// @ts-ignore
+import queryString from 'query-string';
+import { Helmet } from 'react-helmet';
+import { injectT, tType } from '@ndla/i18n';
+import { DefaultErrorMessage } from '../../components/DefaultErrorMessage';
+import { AudioSearch, SearchObject } from '../../interfaces';
+import { podcastSearchQuery } from '../../queries';
+import Podcast from './Podcast';
+
+interface Props {
+  locale: string;
+}
+
+const PodcastListPage: React.FC<Props & tType & RouteComponentProps> = ({
```

Comment:
Vet vi hva ndla ønsker å bruke denne siden til/ use case for siden? 
Trodde først det var for å liste opp podcaster i samme serie, men det er jo ikke noe vi lagrer enda. 
Skal man kunne klikke seg inn til hver av de enkelte podcastene f.eks.?

Reply:
Denne siden er midlertidig fram til gruppering av podcast i serier er klart. Så var denne løsningen Gunnar og jeg kom fram til inntil videre